vllm/scripts.py

```diff
@@ -0,0 +1,193 @@
+# The CLI entrypoint to vLLM.
+import argparse
+import requests
```

Comment:
Let's just `import openai` so you are using this https://github.com/openai/openai-python